packages/@adobe/spectrum-css-temp/components/breadcrumb/index.css

```diff
@@ -121,20 +121,28 @@ governing permissions and limitations under the License.
   pointer-events: none;
 }
 
-.spectrum-Breadcrumbs--compact {
+.spectrum-Breadcrumbs--small {
+  .spectrum-Breadcrumbs-item {
+    font-size: var(--spectrum-breadcrumb-multiline-item-text-size);
+  }
+}
```

Comment:
discarded `font-weight` and `line-height` even on multiline since the `font-weight` is the same regardless and the `line-height` seems to work out just fine